### DISCUSSION TITLE
How to run inference using the inference APIs?

### DISCUSSION THREAD

**Discussion by akshay-k2** [ethvis/qd-track#86] [Q&A]
Hi, 

I'm able to run the test script with the trained model and obtain the inference results for BDD100K. However, I'm unable to figure out how to run inference for a single image in the dataset. 

I see there are inference APIs available in `qdtrack/apis/inference.py`. I'm able to initialize the model by calling `init_model`, however, I'm unsure what to pass as **frame_id** for calling `inference_model(model, imgs, frame_id)`

Any help will be highly appreciated. Thank you.

**Comment by ghost** [comment]
@OceanPang can you please clarify the usage of the inference API?

**Comment by jbohnslav** [comment]
I'm not involved in the development of this project. However, I believe the `frame_id` is just the frame number for those `imgs`. I assumed this on my data with good results.
